proto/beacon/p2p/v1/messages.proto

```diff
@@ -48,5 +48,5 @@ message MetaDataV0 {
 message MetaDataV1 {
   uint64 seq_number = 1;
   bytes attnets = 2 [(ethereum.eth.ext.ssz_size) = "8", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitvector64"];
-  bytes syncnets = 3 [(ethereum.eth.ext.ssz_size) = "64", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitvector512"];
+  bytes syncnets = 3 [(ethereum.eth.ext.ssz_size) = "1", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitvector4"];
```

Comment:
Whats up with this?

Reply:
There are only 4 sync committee subnets(0,1,2,3), previously it was incorrectly set as 512.